app/models/work.rb

```diff
@@ -56,6 +56,11 @@ def self.build_with_empty_version(*args)
     work
   end
 
+  def self.reindex_all
+    all.map { |work| WorkIndexer.call(work) }
```

Comment:
You should use `find_each` here instead of `all`, because under the hood it will automatically load the records out in batches of 1,000 keeping memory usage light. Using `all` is dangerous when tables get large because it can chew through memory and make the server start swapping